### DISCUSSION TITLE
How to use multi-VFO?

### DISCUSSION THREAD

**Discussion by n2iw** [AlexandreRouma/SDRPlusPlus#474] [Q&A] [answered]
This might be a silly question, but how can use multi-VFO? I only see one "radio",  and don't know how to add another.

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
In the Module Manager tab, select 'radio' from the drop-down list, give it a name, and press the '+' button. A new tab should appear with the name you've just given.

![Screenshot from 2021-10-22 20-10-47](https://user-images.githubusercontent.com/41851763/138510572-c7260e5e-71b5-4a66-8801-f98e92a183bd.png)

To be honest, I think for many people this isn't an obvious process, as some might expect adding VFOs to be handled within the Radio tab and don't expect to have to add another Radio module by themselves. @AlexandreRouma do you think there should be a more convenient way to add VFOs?
